assets/scss/settings/fonts.scss

```diff
@@ -0,0 +1,11 @@
+$georgia: Georgia;
+$palatino: Palatino;
+$times_new_roman: Times;
```

Comment:
Name of variable `times_new_roman` should be written in all lowercase letters with hyphens instead of underscores